whenthen-web/src/components/LoginComponent/index.tsx

```diff
@@ -5,13 +5,14 @@ import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
 
 const LoginComponent = () => {
-  const { screenClass, isLogin, changeLoginState } = useRootData(
-    ({ appStore, loginStore }) => ({
+  const { screenClass, setAlert, removeAlert, isLogin, changeLoginState } =
+    useRootData(({ appStore, loginStore }) => ({
       screenClass: appStore.screenClass.get(),
+      setAlert: appStore.setAlert,
+      removeAlert: appStore.removeAlert,
```

Comment:
remove it if you're not going to use it here

Reply:
It will be fixed in the next commit.